Doc/library/functools.rst

```diff
@@ -208,6 +208,55 @@ The :mod:`functools` module defines the following functions:
       Returning NotImplemented from the underlying comparison function for
       unrecognised types is now supported.
 
+.. function:: topsort(pairiter)
```

Comment:
The argument is called "pairlist" in the implementation.  I would use "pairs" in both cases.